roles/apply_nmstate/tasks/main.yml

```diff
@@ -0,0 +1,21 @@
+- name: Install nmstate
+  package:
+    name: nmstate
+    state: present
+
+- name: "Copy nmstate_yaml to {{ nmstate_config_path }}"
```

Comment:
Would `nmstate_rendered_yaml` be a better name for `nmstate_yaml`? As far as I understand, the variable holds a rendered template for _nmstate_.